tox.ini

```diff
@@ -1,20 +1,20 @@
 [tox]
 env_list =
-    py{38,39}-django{42}-{postgresql,sqlite}
-    py{310,311}-django{42,50,51}-{postgresql,sqlite}
-    py{311,312}-django{42,50,51}-{postgresql-psycopg3}
-    py312-django{50,51}-{postgresql-contenttypes}
+    py{38,39}-django{42}-{psycopg2,sqlite}
+    py{310,311}-django{42,50,51}-{psycopg3,psycopg2,sqlite}
+    py312-django{50,51}-{psycopg2}
+    py{312,313}-django{50,51}-{sqlite,psycopg3,postgresql-contenttypes}
 
 [testenv]
 package = wheel
 setenv =
     PYTHONPATH={toxinidir}
-    postgresql: TEST_DB=postgis
-    postgresql: PGUSER=postgres
-    postgresql: PGPASSWORD=postgres
-    postgresql-psycopg3: TEST_DB=postgis
-    postgresql-psycopg3: PGUSER=postgres
-    postgresql-psycopg3: PGPASSWORD=postgres
+    psycopg2: TEST_DB=postgis
```

Comment:
That's a good suggestion, makes it easier to understand. 👍 
However, same thing, I'd love to see this cleanup separately, not in the Python update.

Reply:
For some reason, tox understands that it should install `pyscopg2-binary` when the tox environment is `postgresql-contenttypes`. This causes the tests to break on python 3.13, which is why I had changed the dependency names.

@amureki what do you suggest here?

Reply:
As I see now, tox test matrix is actually confusing, so we would need to improve this.

I filed a reminder, so it would not be forgotten: #506